test/io/test_buffers.cpp

```diff
@@ -68,8 +68,8 @@ class BuffersTest : public ::testing::Test
         memcpy (d.data (), dptr, buffer.size () * sizeof (T));
         buffer.push (d);
         for (size_t j = 0; j < buffer.size (); ++j)
-          if (isnan (eptr[j]))
-            EXPECT_TRUE (isnan (buffer[j]));
+          if (isnan ((double) eptr[j]))
+            EXPECT_TRUE (isnan ((double) buffer[j]));
           else
```

Comment:
I don't like this. ```eptr``` is of template type T and I assume ```buffer``` is something similar. It doesn't make sense to cast them always to double...